### DISCUSSION TITLE
LibreHardwareMonitor for Rainmeter

### DISCUSSION THREAD

**Discussion by bratskij** [LibreHardwareMonitor/LibreHardwareMonitor#555] [Ideas]
Hey. I write with a translator, so that immediately apologize. Is there any Plugins for Rainmeter? OpenHardwareMonitor for this is, but it is not suitable for LibreHardwareMonitor. It is possible to do something new or edit an old one?
[OpenHardwareMonitorPlugin.zip](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/files/3955634/OpenHardwareMonitorPlugin.zip)

**Comment by radhaz** [comment]
I'm another person eager to use this in conjuction with Rainmeter. I have "Sidebar Diagnostics" for the time being but the way the data is presented in that tool isn't as customizable as Rainmeter.

**Comment by black-ish** [comment]
Oh I would love to see that too.

**Comment by nitehu** [comment]
Hello guys,
I've started a project to provide a rough LibreHardwareMonitor Plugin for Rainmeter. You may [check it out here](https://github.com/nitehu/Rainmeter-LibreHardwareMonitor).
Please let me know if it is working for you at all, or you have any problems with it.

**Comment by darkbrain-fc** [comment]
Maybe you can use this: https://github.com/darkbrain-fc/HardwareSupervisorRainmeterPlugin Now latest HardwareSupervisor https://github.com/darkbrain-fc/HardwareSupervisor has migrated to LibreHardwareMonitor

**Comment by MagFlux** [comment]
I realize this is originally an ancient post but it appears there's a pretty active/recent repo that creates a OHM/LHM plugin for Rainmeter here: [https://github.com/abichinger/Rainmeter-HardwareMonitor](https://github.com/abichinger/Rainmeter-HardwareMonitor).
